### PR TITLE
Feat: Added work experience end date tooltip

### PR DESCRIPTION
**Pull request by by-bai**
# Notion Task

https://www.notion.so/Work-Experience-End-Date-Tooltip-fb27d00fae3a428f82db8957bae7737c

# Changes Made In This PR

- Modified Input component to include "tooltip" prop for rendering tooltip.
- Added tooltip for End Date

# Screenshots

**Before:**
Screenshot 2021-12-30 at 12 17 41 AM

**After:**
Screenshot 2021-12-30 at 12 15 06 AM

### CODE REVIEW

File: src/components/shared/Input.js [hunk 1]
```diff
@@ -54,13 +56,20 @@ const Input = ({
   return (
     <div className={className}>
       <label htmlFor={uuid}>
-        <span>
+        <span className="flex flex-row gap-1 items-center">
```
Comment (millasml) [inline src/components/shared/Input.js:59]: remove `items-center`

File: src/components/shared/Input.js [hunk 2]
```diff
@@ -54,13 +56,20 @@ const Input = ({
...
           {isRequired && (
             <span className="opacity-75 font-normal lowercase">
               ({t('shared.forms.required')})
             </span>
           )}
+          {tooltip && (
+            <Tooltip title={tooltip} placement="right" arrow>
+              <div>
+                <MdInfoOutline size={12} />
+              </div>
```
Comment (millasml) [inline src/components/shared/Input.js:70]: remove the `<div>` wrapping MdInfoOutline

**Review by millasml — CHANGES_REQUESTED** [review]
a few more changes after a discussion in https://github.com/CareerSocius/resumebuilder/pull/29#discussion_r777111658

**Comment by by-bai** [comment]
Made changes as requested and rebased branch!

After:
Screenshot 2022-01-02 at 4 47 15 PM

**Review by millasml — APPROVED** [review]
LGTM!

**Pull request merged**